### PR TITLE
Introduce new namespace and solve duplicate key issue

### PR DESCRIPTION
**Pull request by smoes**
…4 by default but can be overridden by user.
Fixes issue #780 and allows using the newest Wix toolset.

### CODE REVIEW

**Comment by smoes** [comment]
The error is due to the standard major version being `4`, but it is using `3`. Please add `wixMajorVersion := 3` to settings.

File: src/main/scala/com/typesafe/sbt/packager/windows/WindowsPlugin.scala [hunk 1]
```diff
@@ -57,6 +57,8 @@ object WindowsPlugin extends AutoPlugin {
       "-cultures:en-us"),
     wixProductId := WixHelper.makeGUID,
     wixProductUpgradeId := WixHelper.makeGUID,
+    wixMajorVersion := 4,
```
Comment (muuki88) [inline src/main/scala/com/typesafe/sbt/packager/windows/WindowsPlugin.scala]: Can you change this to `3`, so we remain backwards compatible and the tests run.

**Comment by muuki88** [comment]
Code looks good. Please add some [documentation](http://sbt-native-packager.readthedocs.io/en/latest/formats/windows.html). You can build the docs [with this guide](https://github.com/sbt/sbt-native-packager/wiki/Developer-Guide#documentation)

Also an additional windows test would be nice, it seems appveyor only [supports Wix 3.10](https://www.appveyor.com/docs/installed-software#languages-libraries-frameworks).

**Comment by smoes** [comment]
Now also resolves issue #781. 
I removed the auto start menu shortcut generation for config files, since that should be left to the user anyways. That resolves the conflict of two coexisting AddShortCut WindowsFeatures. Config files still can be added to the start menu manually.

**Comment by smoes** [comment]
So the final solution for #781 was different than I expected:

The RemoveFolder directive has nothing to do with shortcuts. I re-inserted the previous standard shortcuts for the menu files but removed the RemoveFolder directive for ApplicationFolder from the AddShortcuts xml. It was moved to ApplicationFolder itself so that it appears only once in the very right spot, independent from shortcuts.

**Comment by muuki88** [comment]
LGTM. I will try t o test it this week on my windows machine.

@nazoking do you have time to test this as well?

**Pull request merged**